src/main/java/org/raystack/depot/metrics/StatsDReporter.java

```diff
@@ -69,7 +69,7 @@ private String[] getTags(String[] tags) {
             return null;
         }
         List<String> list = Arrays.stream(tags).map(s -> s.replaceAll("=", ":")).collect(Collectors.toList());
-        list.add(this.getGlobalTags());
+        list.addAll(Arrays.stream(this.getGlobalTags().split(",")).collect(Collectors.toList()));
```

Comment:
Instead of doing stream, we can do `list.addAll(Arrays.asList(this.getGlobalTags().split(",")));`

Reply:
changed